src/view/UserphraseView.cpp

```diff
@@ -71,14 +79,18 @@ void UserphraseView::addPhrase(int result)
     qDebug() << "Add" << *phrase.get() << "(" << *bopomofo.get() << ")";
 
     if (dialogType_ == DIALOG_MODIFY) {
-        remove();
+        remove(QDialog::Accepted);
```

Comment:
I'd like to know how to better implement this, too. If you can, please give me some thoughts.